src/kemal/init_handler.cr

```diff
@@ -9,6 +9,7 @@ module Kemal
     def call(context : HTTP::Server::Context)
       context.response.headers.add "X-Powered-By", "Kemal" if Kemal.config.powered_by_header?
       context.response.content_type = "text/html" unless context.response.headers.has_key?("Content-Type")
+      context.response.headers.add "Date", Time.utc.to_rfc2822
```

Comment:
Should this use https://crystal-lang.org/api/HTTP.html#format_time%28time%3ATime%29%3AString-class-method instead? This would properly use `GMT` vs `+0000` as the timezone.